dotCMS/src/main/java/com/dotcms/rest/ContentResource.java

```diff
@@ -794,7 +813,11 @@ private Response multipartPUTandPOST(HttpServletRequest request, HttpServletResp
                         String fieldName = ff.getFieldContentlet();
                         if (fieldName.startsWith("binary")
                                 && !usedBinaryFields.contains(fieldName)) {
-                            contentlet.setBinary(ff.getVelocityVarName(), tmp);
+                            String fieldVarName = ff.getVelocityVarName();
+                            if (binaryFields.size() > 0) {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Use isEmpty() to check whether the collection is empty or not. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1155)